distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/tomcat/catalina-server.xml.j2

```diff
@@ -59,6 +59,11 @@
                        prefix="http_access_" suffix=".log" pattern="%h %l %u %t %r %s %b %{Referer}i %{User-Agent}i %T"/>
                 <Valve className="org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve" threshold="600"/>
                 <Valve className="org.wso2.carbon.tomcat.ext.valves.CompositeValve"/>
+
+                <!-- Authentication and Authorization valve for the rest apis and we can configure context for this in identity.xml.j2  -->
+                <Valve className="org.wso2.carbon.identity.auth.valve.AuthenticationValve"/>
+                <Valve className="org.wso2.carbon.identity.authz.valve.AuthorizationValve"/>
+                <Valve className="org.wso2.carbon.identity.context.rewrite.valve.TenantContextRewriteValve"/>
```

Comment:
maintain this change on product-is repo.

Reply:
Fixed.